### PR TITLE
Added a super simple PR template

### LINKED ISSUE #6: Add a Pull Request Template

### PR DESCRIPTION
**Pull request by soygabimoreno**

### CODE REVIEW

**Review by RubenGonz2 — COMMENTED** [review]
Once we have this template, how can you use it in a PR?

It's only a copy/paste template, or you can use it directly during a PR creation?

**Comment by soygabimoreno** [comment]
> Once we have this template, how can you use it in a PR?
> 
> It's only a copy/paste template, or you can use it directly during a PR creation?

It appears directly.
This is the cool thing.

**Review by joseantoniocampillo — APPROVED** [review]
Very good!

**Comment by soygabimoreno** [comment]
I would need another approval to merge this PR.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+### :tophat: What is the goal?
+
```
Comment (aflopezbec) [inline .github/PULL_REQUEST_TEMPLATE.md:2]: We could added more sections as:

## Aditional information


## Types of changes




- [ ] Docs change / refactoring / dependency upgrade.
- [ ] Bug fix.
- [ ] New feature.
- [ ] Improvement.

**Review by aflopezbec — APPROVED** [review]

**Pull request merged**